share/instruments/instruments.xml

```diff
@@ -5805,15 +5805,15 @@
             <Instrument id="bass-drum">
                   <family>drums</family>
                   <longName>Concert Bass Drum</longName>
-                  <shortName>Con. BD </shortName>
+                  <shortName>Con. BD</shortName>
```

Comment:
This change here requires an updated intrumentsxml.h, like by running the generateTs.py script.

Reply:
Good point. I will run the translation script.

Reply:
Actually, I won't bother running the script because it won't work for much longer anyway due to other changes coming in future PRs.